modular_zubbers/code/modules/loadouts/overrides/loadout_items/loadout_datum.dm

```diff
@@ -69,3 +69,37 @@
 /datum/loadout_item/under/jumpsuit/utility_sec/New()
 	. = ..()
 	LAZYOR(restricted_roles,JOB_SECURITY_MEDIC)
```

Comment:
Whilst we're here, you'll make me very happy if you redo these back to
```suggestion
/datum/loadout_item/under/jumpsuit/utility_sec/New()
	. = ..()
	restricted_roles |= JOB_SECURITY_MEDIC
```

because I forgot 3 months ago

Reply:
Sure gimme two minutes.